### PR TITLE
Select destination country

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/screens/Home/styles.scss [hunk 1]
```diff
@@ -18,46 +15,37 @@
...
-    background-color: $whitesmoke;
-  }
-}
+ .main__button {
+   display: flex;
+   position: absolute;
+   justify-content: center;
+   align-items: center;
+   align-self: center;
+   padding-top: 350px;
```
Comment (mi-chal) [inline src/screens/Home/styles.scss]: Why is this padding so high?

File: src/screens/Home/styles.scss [hunk 2]
```diff
@@ -18,46 +15,37 @@
...
-//   flex-direction: column;
-//   position: absolute;
-
-// }
-
-.main__button {
+.main__button-search {
   display: flex;
   width: 100px;
   position: absolute;
```
Comment (mi-chal) [inline src/screens/Home/styles.scss]: Why it has to be absolute?

File: src/screens/Home/styles.scss (hunk 1 above)
Comment (dwantula) [reply src/screens/Home/styles.scss]: Fixed

File: src/screens/Home/styles.scss (hunk 2 above)
Comment (dwantula) [reply src/screens/Home/styles.scss]: Fixed

**Pull request merged**